R/client-connect.R

```diff
@@ -59,6 +59,7 @@ connectClient <- function(service, authInfo) {
       application <- POST_JSON(service, authInfo, "/applications", details)
       list(
         id = application$id,
+        guid = application$guid,
```

Comment:
Need same change for other clients?

Reply:
I don't think so. Only connect has `guid`. See the set of fields that are "kept" from API list responses in `applications()`.